x-pack/docs/en/settings/monitoring-settings.asciidoc

```diff
@@ -85,10 +89,6 @@ You can update this setting through the
 
 Sets the timeout for collecting index statistics. Defaults to `10s`.
 
-`xpack.monitoring.collection.indices.stats.timeout`::
```

Comment:
Seems this has already been removed meanwhile in https://github.com/elastic/elasticsearch/pull/32229. Can you please merge master and resolve the conflict? Then all should be fine.